db/migrate/20160308143022_update_feedback_author_id_based_on_cloned_from.rb

```diff
@@ -0,0 +1,25 @@
+class UpdateFeedbackAuthorIdBasedOnClonedFrom < ActiveRecord::Migration
+  def up
+    ActiveRecord::Base.transaction do
+      cloned_people = select_all("
+         SELECT id, cloned_from, community_id FROM people WHERE cloned_from IS NOT NULL
+      ").to_ary
+      cloned_people.each { |p|
+        execute("
+          UPDATE feedbacks SET author_id = #{quote(p['id'])}
+          WHERE community_id = #{quote(p['community_id'])} AND author_id = #{quote(p['cloned_from'])}
+        ")
```

Comment:
This is now looping through all cloned users. I'm wondering, could we implement this with only one SQL query?

```sql
UPDATE feedbacks
LEFT JOIN people ON people.cloned_from = feedbacks.author_id AS cloned_people
SET feedbacks.author_id = cloned_people.id
WHERE feedbacks.author_id = cloned_people.cloned_from AND feedbacks.community_id = cloned_people.community_id
```